server/apis/v1/handler.go

```diff
@@ -1684,3 +1684,24 @@ func (h *handler) getContainerStatus(state corev1.ContainerState) string {
 		return "Unknown"
 	}
 }
+
+func cpuToMillicores(quantityStr string) string {
```

Comment:
Why don't take `Quantity` directly so that there's no need to parse?

Reply:
[Quantity](https://pkg.go.dev/k8s.io/apimachinery@v0.31.2/pkg/api/resource#Quantity) is like:
`` <quantity> ::= <signedNumber><suffix>. To get the suffix and calculate accordingly we're parsing the string.
For eg: 2847159n this is the cpuQuantity, `cpuQuantity.MilliValue()` will give us 3, but we want 2.84m as our final result. For that we divide 2847159/1e6